### PR TITLE
feat: rabbitmq assertions

### PR DESCRIPTION
**Pull request by f0rmiga**
The RabbitMQ assertions in this PR depend on the changes from https://github.com/kubernetes-sigs/minibroker/pull/97.

### CODE REVIEW

File: assets/rabbitmqapp/go.mod [hunk 1]
```diff
@@ -0,0 +1,18 @@
+module github.com/SUSE/minibroker-integration-tests/assets/mariadbapp
```
Comment (mook-as) [inline assets/rabbitmqapp/go.mod]: Wait, this isn't `mariadbapp`. I must have missed the same in the other PRs… sorry.

**Review by mook-as — CHANGES_REQUESTED** [review]

**Review by mook-as — DISMISSED** [review]
I'm fine with fixing `go.mod` in a separate PR.

File: assets/rabbitmqapp/go.mod (hunk 1 above)
Comment (f0rmiga) [reply assets/rabbitmqapp/go.mod]: Busted!

Comment (f0rmiga) [reply assets/rabbitmqapp/go.mod]: Fixed with https://github.com/SUSE/minibroker-integration-tests/pull/8/commits/441a85cba1f47826c83a099c190c0d50f848532a.

**Pull request merged**